apps/src/StudioApp.js

```diff
@@ -2172,6 +2188,13 @@ StudioApp.prototype.handleHideSource_ = function(options) {
   }
 };
 
+StudioApp.prototype.handleIframeEmbedAppAndCode_ = function() {
+  document.body.style.backgroundColor = 'transparent';
+  document.body.className += 'iframe_embed_app_and_code';
```

Comment:
You're appending to `className` here but not supplying whitespace, so if body has an existing classname such as `"responsive dark-mode"` this line would change it to `"responsive dark-modeiframe_embed_app_and_code"` which won't work.  You could add spaces on each side of your new class name, or the newer way to do this (which [should work on all browsers we support](https://caniuse.com/#feat=classlist)) is
```suggestion
  document.body.classList.add('iframe_embed_app_and_code');
```